docker-compose.open-solver.yml

```diff
@@ -0,0 +1,4 @@
+version: "3.6"
+services:
+  stablex:
+    env_file: common-open-solver.env
```

Comment:
I think this file should also specify the image. This way you shouldn't need to differentiate between docker-compose.binary.public/private.yml but can instead keep a single docker-compose.binary.yml .

Also does exposing the `SOLVER_TYPE` you need inside `environment` not work? This way you wouldn't need the new docker-compose.binary.private.yml file